stock_pack_operation_auto_fill/__manifest__.py

```diff
@@ -1,18 +1,22 @@
 # Copyright 2017 ACSONE SA/NV
+# Copyright 2018 Tecnativa - Sergio Teruel
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     'name': 'Stock Pack Operation Auto Fill',
     'summary': """
         Stock pack operation auto fill""",
-    'version': '11.0.1.0.1',
+    'version': '11.0.2.0.0',
     'license': 'AGPL-3',
-    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'author': 'ACSONE SA/NV,'
+              'Tecnativa,'
+              'Odoo Community Association (OCA)',
     'website': 'https://acsone.eu/',
```

Comment:
@sergio-teruel Improve URL ser per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)